packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -118,12 +118,14 @@ export async function generateCrudInput(
             const initializer = morphTsProperty(prop.name, metadata).getInitializer()?.getText();
             const defaultValue = prop.nullable && (initializer == "undefined" || initializer == "null") ? "null" : initializer;
             const fieldOptions = tsCodeRecordToString({ nullable: prop.nullable ? "true" : undefined, defaultValue });
+            let returnTypeFunction = ``;
             if (integerTypes.includes(prop.columnTypes[0])) {
                 decorators.push("@IsInt()");
+                returnTypeFunction = `() => Int, `;
             } else {
                 decorators.push("@IsNumber()");
             }
-            decorators.push(`@Field(${fieldOptions})`);
+            decorators.push(`@Field(${returnTypeFunction}${fieldOptions})`);
```

Comment:
I tried to update the generate-crud-input-integer.spec.ts but I found the decorator arguments fields empty.